cmd/runtimetest/main.go

```diff
@@ -271,6 +271,62 @@ func validateDefaultFS(spec *rspec.Spec) error {
 	return nil
 }
 
+func validateLinuxDevices(spec *rspec.Spec) error {
+	logrus.Debugf("validating linux devices")
+
+	for _, device := range spec.Linux.Devices {
+		fi, err := os.Stat(device.Path)
+		if err != nil {
+			return err
+		}
```

Comment:
Perhaps not technically true (e.g. you could remove the device before calling `start`), but probably true for all the configurations we'll run in our validation tests.  Do we want to document that caveat somewhere?  Maybe add:

    app.Description = "runtimetest compares its current environment with an OCI runtime configuration read from `config.json` in its current working directory.  The tests are fairly generic and cover most configurations used by the runtime validation suite, but there are corner cases where a container launched by a valid runtime would not satisfy runtimetest."

to `main`?